lib/srv/termhandlers.go

```diff
@@ -212,3 +265,31 @@ func parseWinChange(req *ssh.Request) (*rsession.TerminalParams, error) {
 	params, err := rsession.NewTerminalParamsFromUint32(r.W, r.H)
 	return params, trace.Wrap(err)
 }
+
+func parseFileTransferRequest(req *ssh.Request) (*rsession.FileTransferParams, error) {
+	var r sshutils.FileTransferReqParams
+	if err := ssh.Unmarshal(req.Payload, &r); err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	params := &rsession.FileTransferParams{
+		Location:  r.Location,
+		Filename:  r.Filename,
+		Size:      r.Size,
+		Direction: r.Direction,
```

Comment:
Is this checked anywhere to ensure it's either `upload` or `download`?

Reply:
I switched to an enum in the file transfer event and didnt switch here by mistake. I'm going to convert this to an enum so we won't have to do the check